registry/polkadot.json

```
@@ -31,4 +31,13 @@
         "longDescription": "## About Sik\nFounder of [Crifferent Staking Services](https://crifferent.de) \n\nDeveloper for [Staking Facilities](https://stakingfacilities.com) \n\nMember of [DOT Validator Alliance DAO](https://dotvalidators.org) \n\nTeam member @[HydraDX](https://hydradx.io)/[Basilisk](https://bsx.fi) \n\nRPC provider @[UnitedBloc](https://unitedbloc.com/) \n\n---\n\n**From within the ecosystem - for the ecosystem!** \n\nI'm evaluating proposals by taking different views into consideration. I'm not only looking at it from the perspective of a community member, but also as a developer/founder/infra-provider/human being. \n\n---\n## Contact me \n\n@dev0_sik @[Twitter](https://twitter.com/dev0_sik) \n\n@dev0_sik @[Telegram](https://t.me/dev0_sik) \n\nsik#3706 @[Discord](https://discord.com/) \n\n@dev0_sik:matrix.org @[Element](https://element.io/) \n\n[simon.kraus@crifferent.de](mailto:simon.kraus@crifferent.de) \n\nWhatsApp/Callboy +49 157 377 309 02",
         "isOrganization": false
     }
-]
+     },
+    {
+        "address": "12mP4sjCfKbDyMRAEyLpkeHeoYtS5USY4x34n9NMwQrcEyoh",
+        "name": "PolkaWorld",
+        "image": "https://github.com/pppxj1/opengov-delegate-registry/blob/master/images/PolkaWorld.png",
+        "shortDescription": "the largest Polkadot Chinese community in Asia",
+        "longDescription": "## About us\PolkaWorld is the largest Polkadot Chinese community in Asia and has formerly served as a Council Member for Polkadot and Kusama. It has been involved in numerous proposal votings and has a profound understanding of Polkadot's OpenGov governance system",
+        "isOrganization": true
+    }
+
```

Hello @pppxj1, thanks for request!
There is some fixes which should be done, before we will be able to merge your request:
```suggestion
     ,
    {
        "address": "12mP4sjCfKbDyMRAEyLpkeHeoYtS5USY4x34n9NMwQrcEyoh",
        "name": "PolkaWorld",
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/PolkaWorld.png",
        "shortDescription": "the largest Polkadot Chinese community in Asia",
        "longDescription": "## About us\n\nPolkaWorld is the largest Polkadot Chinese community in Asia and has formerly served as a Council Member for Polkadot and Kusama. It has been involved in numerous proposal votings and has a profound understanding of Polkadot's OpenGov governance system",
        "isOrganization": true
    }
]
```